mediacapture-streams/obtaining-local-multimedia-content/navigatorusermedia/deny.html

```diff
@@ -20,16 +20,18 @@ <h1 class="instructions">Description</h1>
 <script>
 var t = async_test("Tests that the error callback is triggered when permission is denied", {timeout:10000});
 t.step(function() {
-  navigator.getUserMedia({video: true}, t.step_func(function (stream) {
-    assert_unreached("The success callback should not be triggered since access is to be denied");
-    t.done();
-  }),
-  t.step_func(function (error) {
-    assert_equals(error.name, "securityError", "securityError returned");
-    assert_equals(error.constraintName, undefined, "constraintName attribute not set for permission denied");
-    };
-    t.done();
-  }));
+  navigator.getUserMedia(
+    {video: true},
+    t.step_func(function (stream) {
+      assert_unreached("The success callback should not be triggered since access is to be denied");
+      t.done();
+    }),
+    t.step_func(function (error) {
+      assert_equals(error.name, "securityError", "securityError returned");
+      assert_equals(error.constraintName, undefined, "constraintName attribute not set for permission denied");
+     t.done();
```

Comment:
the three arguments of getUserMedia are at the same level of indentation. I
don't understand the problem.

On Wed, Mar 23, 2016 at 2:37 PM, Josh Matthews <notifications@github.com>
wrote:

> In
> mediacapture-streams/obtaining-local-multimedia-content/navigatorusermedia/deny.html
> <https://github.com/w3c/web-platform-tests/pull/2723#discussion_r57241704>
> :
>
> > -  t.step_func(function (error) {
> > -    assert_equals(error.name, "securityError", "securityError returned");
> > -    assert_equals(error.constraintName, undefined, "constraintName attribute not set for permission denied");
> > -    };
> > -    t.done();
> > -  }));
> > +  navigator.getUserMedia(
> > +    {video: true},
> > +    t.step_func(function (stream) {
> > +      assert_unreached("The success callback should not be triggered since access is to be denied");
> > +      t.done();
> > +    }),
> > +    t.step_func(function (error) {
> > +      assert_equals(error.name, "securityError", "securityError returned");
> > +      assert_equals(error.constraintName, undefined, "constraintName attribute not set for permission denied");
> > +     t.done();
>
> Indent this to match, please?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/w3c/web-platform-tests/pull/2723/files/f743a4ef3c3ae66102e46b8dc5e3eb3bdbfd07a8#r57241704>
>



-- 
Alex. Gouaillard, PhD, PhD, MBA
------------------------------------------------------------------------------------
Principal Architect - Citrix, San Francisco
President - CoSMo Software Consulting, Singapore
------------------------------------------------------------------------------------
sg.linkedin.com/agouaillard

   -